embrace-android-sdk/src/androidTest/assets/golden-files/session-end.json

```diff
@@ -105,5 +105,6 @@
     "un": "John Doe",
     "per": ["first_day"]
   },
+  "spans": "__EMBRACE_TEST_IGNORE__",
```

Comment:
Adding validation in next PR when the issue of double-logging fixed if the moment ends in a timeout